docs/tutorials/stateful-application/basic-stateful-set.md

```diff
@@ -0,0 +1,728 @@
+demonstrate the basic features of a StatefulSet, and to not conflate the former 
+topic with the latter, you will deploy a simple web application using StatefulSets.
+
+After this tutorial, you will be familiar with the following.
+
+* How to create a StatefulSet
+* How a StatefulSet manages its Pods
+* How to delete a StatefulSet
+* How to scale a StatefulSet
+* How to update the container image of a StatefulSet's Pods
+{% endcapture %}
+
+{% capture lessoncontent %}
+### Creating a StatefulSet 
+
+Begin by creating a StatefulSet using the example below. It is similar to the 
+example presented in the
+[StatefulSets](/docs/concepts/controllers/statefulsets/) concept. It creates 
+a [Headless Service](/docs/user-guide/services/#headless-services), `nginx`, to 
+control the domain of the StatefulSet, `web`. 
+
+{% include code.html language="yaml" file="web.yaml" ghlink="/docs/tutorials/stateful-application/web.yaml" %}
+
+Download the example above, and save it to a file named `web.yaml`
+
+You will need to use two terminal windows. In the first terminal, use 
+[`kubectl get`](/docs/user-guide/kubectl/kubectl_get/) to watch the creation 
+of the StatefulSet's Pods.
+
+```shell
+kubectl get pods -w -l app=nginx
+```
+
+In the second terminal, use 
+[`kubectl create`](/docs/user-guide/kubectl/kubectl_create/) to create the 
+Headless Service and StatefulSet defined in `web.yaml`.
+
+```shell
+kubectl create -f web.yml 
+service "nginx" created
+statefulset "web" created
+```
+
+The command above creates two Pods, each running an 
+[NGINX](https://www.nginx.com) webserver. Get the `nginx` Service and the 
+`web` StatefulSet to verify that they were created successfully.
+
+```shell
+kubectl get service nginx
+NAME      CLUSTER-IP   EXTERNAL-IP   PORT(S)   AGE
+nginx     None         <none>        80/TCP    12s
+
+kubectl get statefulset web
+NAME      DESIRED   CURRENT   AGE
+web       2         1         20s
+```
+
+#### Ordered Pod Creation
+
+For a StatefulSet with N replicas, when Pods are being deployed, they are 
+created sequentially, in order from {0..N-1}. Examine the output of the 
+`kubectl get` command in the first terminal. Eventually, the output will 
+look like the example below.
+
+```shell
+kubectl get pods -w -l app=nginx
+NAME      READY     STATUS    RESTARTS   AGE
+NAME      READY     STATUS    RESTARTS   AGE
+web-0     0/1       Pending   0          0s
+web-0     0/1       Pending   0         0s
+web-0     0/1       ContainerCreating   0         0s
+web-0     1/1       Running   0         19s
+web-1     0/1       Pending   0         0s
+web-1     0/1       Pending   0         0s
+web-1     0/1       ContainerCreating   0         0s
+web-1     1/1       Running   0         18s
+```
+
+Notice that the `web-0` Pod is launched and set to Pending prior to 
+launching `web-1`. In fact, `web-1` is not launched until `web-0` is 
+[Running and Ready](/docs/user-guide/pod-states). 
+
+### Pods in a StatefulSet
```

Comment:
Try not to nest headings directly under one another. That's usually indicative of structural problems or a missing introductory sentence. In this case it's the latter.

"Unlike Pods in other controllers, the Pods in a StatefulSet have a unqiue ordinal index and a stable network identity."